### PR TITLE
lib/model: Trigger a pull when ignore patterns change

### PR DESCRIPTION
**Pull request by calmh**
### Purpose

Improving quality of life

### CODE REVIEW

**Comment by imsodin** [comment]
@st-review merge

**Comment by st-review** [comment]
@imsodin: Build status is `pending`. I'll wait until it goes green and then merge!

File: lib/model/folder.go [hunk 1]
```diff
@@ -50,8 +50,6 @@ func (f *folder) DelayScan(next time.Duration) {
 	f.scan.Delay(next)
 }
 
-func (f *folder) IndexUpdated() {}
-
 func (f *folder) IgnoresUpdated() {
```
Comment (AudriusButkevicius) [inline lib/model/folder.go]: There should be a concrete implementation somewhere, as well as mocks in tests etc?

Comment (calmh) [reply lib/model/folder.go]: This was literally the last remains of IndexUpdated.

**Comment by st-review** [comment]
:ok_hand: Merged as 0901350087496d441a3e0ae68c7b0dbe179e2f82. Thanks, @calmh!

**Pull request closed without merging**